pages/add-clinic.vue

```diff
@@ -113,8 +115,17 @@ export default {
       (v) => !!v || "Website URL is required",
       (v) => (v && v.length >= 5) || "Please enter a valid URL",
     ],
-  }),
 
+    //SnackBar Notification Data
+    snackBarData: {
```

Comment:
this should be part of the Vue component `data` for proper reactivity and follow vue local state management guidelines.
See: https://v3.vuejs.org/api/options-data.html

Reply:
Im sorry i thought i added it to the data component 
i lost track of the brackets opening and closing ill do the necessary changes